datamade/la-metro-councilmatic
# Omit meetings without minutes from Minutes page

## Overview

This PR modifies the Minutes page to only list meetings that can have minutes, i.e. meetings from the board, LA SAFE, and Crenshaw.

### Checklist



## Testing Instructions

 * Navigate to the Minutes page
 * Verify that the page only displays meetings from the board, LA SAFE, and Crenshaw

Handles #808
